helpers/entradaHelper/entrada.helper.go

```diff
@@ -277,7 +283,7 @@ func AprobarEntrada(entradaId int) (result map[string]interface{}, outputError m
 	var groups = make(map[int]float64)
 	i := 0
 	for _, elemento := range transaccionActaRecibido.Elementos {
-		fmt.Println("entra:")
+		logs.Debug("entra:", elemento.SubgrupoCatalogoId)
```

Comment:
Comentar
```suggestion
		// logs.Debug("entra:", elemento.SubgrupoCatalogoId)
```

o Eliminar
```suggestion
```